modP2pImpl/src/org/aion/p2p/impl1/P2pMgr.java

```diff
@@ -469,7 +470,8 @@ private TaskReceive getReceiveInstance() {
     }
 
     private TaskStatus getStatusInstance() {
-        return new TaskStatus(this.nodeMgr, this.selfShortId, this.sendMsgQue, this.receiveMsgQue);
+        return new TaskStatus(
+                this.start, this.nodeMgr, this.selfShortId, this.sendMsgQue, this.receiveMsgQue);
```

Comment:
I think we could omit all the `this.` and keep the code in one line

Reply:
updated